### PR TITLE
tests: Use a /cache directory instead of /images

### PR DESCRIPTION
**Pull request by stefwalter**
Also parameterize the place to publish to.

### CODE REVIEW

**Comment by stefwalter** [comment]
@dperpeet This has been built into a container, pushed and deployed to the openshift nodes and falcon.

**Comment by stefwalter** [comment]
Rebased.

File: tests/Dockerfile [hunk 1]
```diff
@@ -74,17 +74,17 @@ ENV TMPDIR /build/tmp
...
 
 VOLUME /secrets
-VOLUME /images
+VOLUME /cache
 
 USER user
 WORKDIR /build
-ENTRYPOINT ["/usr/local/bin/cockpit-tests"]
-CMD ["--publish=fedorapeople.org", "--verbose"]
+CMD /usr/local/bin/cockpit-tests --publish=$TEST_PUBLISH --verbose
```
Comment (martinpitt) [inline tests/Dockerfile]: Thanks for this, this is much  less confusing.

File: tests/install-service [hunk 2]
```diff
@@ -3,24 +3,25 @@
...
+mkdir -p $SECRETS $CACHE
+chown -R 1111:1111 $SECRETS $CACHE
 
 cat <<EOF > /etc/systemd/system/cockpit-tests.service
 [Unit]
 Description=Cockpit Verify
-Requires=docker.service libvirtd.service
-After=docker.service libvirtd.service
+Requires=docker.service
+After=docker.service
```
Comment (martinpitt) [inline tests/install-service]: Ah, nice cleanup along the way, now the libvirt stuff is happening inside the docker container (through vm-prep).

**Pull request merged**